source/hic_hal/stm32/stm32h743ii/IO_Config.h

```diff
@@ -29,51 +29,46 @@
 COMPILER_ASSERT(DAPLINK_HIC_ID == DAPLINK_HIC_ID_STM32H743II);
 
 //USB control pin
-#define USB_CONNECT_PORT_ENABLE()    __HAL_RCC_GPIOA_CLK_ENABLE()
-#define USB_CONNECT_PORT_DISABLE()   __HAL_RCC_GPIOA_CLK_DISABLE()
-#define USB_CONNECT_PORT             GPIOA
-#define USB_CONNECT_PIN              GPIO_PIN_15
-#define USB_CONNECT_ON()             (USB_CONNECT_PORT->BSRR = USB_CONNECT_PIN)
-#define USB_CONNECT_OFF()            (USB_CONNECT_PORT->BSRR  = ((uint32_t)USB_CONNECT_PIN << 16))
+#define USB_CONNECT_PORT_ENABLE()       __HAL_RCC_GPIOA_CLK_ENABLE()
+#define USB_CONNECT_PORT_DISABLE()      __HAL_RCC_GPIOA_CLK_DISABLE()
+#define USB_CONNECT_PORT                GPIOA
+#define USB_CONNECT_PIN                 GPIO_PIN_15
+#define USB_CONNECT_ON()                (USB_CONNECT_PORT->BSRR = USB_CONNECT_PIN)
+#define USB_CONNECT_OFF()               (USB_CONNECT_PORT->BSRR  = ((uint32_t)USB_CONNECT_PIN << 16))
 
 //Connected LED
-
-//#define CONNECTED_LED_PORT           GPIOA		//elee: LED1 on eval board
-//#define CONNECTED_LED_PIN            GPIO_PIN_4  //elee: LED1 on eval board
-//#define CONNECTED_LED_PIN_Bit        4
-
-#define CONNECTED_LED_PORT           GPIOG		//elee: LED0 on UDB board
-#define CONNECTED_LED_PIN            GPIO_PIN_0  //elee: LED0 on UDB board
-#define CONNECTED_LED_PIN_Bit        0
+#define CONNECTED_LED_PORT              GPIOG
+#define CONNECTED_LED_PIN               GPIO_PIN_0
+#define CONNECTED_LED_PIN_Bit           0
 
 // ToDo(elee):
```

Comment:
What's this TODO about? Is it still relevant?

Reply:
It can go.  I believe the thought was when the UDB was in bootloader mode it might try to turn off power to the DUT (thru the downstream USB port power).  
But I don't see a strong reason to do that.  We should rarely be in udb_bootloader, and I can't think of what actual harm it would do.